apps/web/app/(booking-page-wrapper)/[user]/[type]/page.tsx

```diff
@@ -31,7 +31,7 @@ export const generateMetadata = async ({ params, searchParams }: PageProps) => {
       })),
     ],
   };
-  const decodedParams = decodeParams(params);
+  const decodedParams = decodeParams(await params);
```

Comment:
Is this a promise now?

Reply:
yup! https://nextjs.org/docs/messages/sync-dynamic-apis